quic/s2n-quic-core/src/datagram/default.rs

```diff
@@ -49,25 +208,20 @@ impl Sender for DefaultSender {
     }
 }
 
-impl DefaultSender {
-    /// Creates a builder for the default datagram sender
-    pub fn builder() -> Builder {
-        Builder::default()
-    }
-}
-
 /// A builder for the default datagram sender
 ///
-/// Use to configure a datagram expiration time and send queue size
+/// Use to configure a datagram send queue size
 #[derive(Debug)]
 pub struct Builder {
     queue_capacity: usize,
+    connection_info: ConnectionInfo,
```

Comment:
I wouldn't store the actual struct here, just the information you care about, which is the peer's datagram size limit.